lua/entities/gmod_wire_expression2/core/http.lua

```diff
@@ -101,4 +126,4 @@ end
 
 registerCallback( "destruct", function( self )
 	run_on.ents[self.entity] = nil
-end )
+end )
```

Comment:
For what it's worth, the "Sorry about that whitespace" change actually *broke* the whitespace on this line when you'd previously not touched it, but that's okay - I can take it out and merge manually.

Reply:
Sorry about that, I'll revert this.